translation for trusted committer chapter 02 and 03 to german added

RJPlog opened the pull request:
I finished Translation for chapter 02 and 03 of trusted committer section, compare #256

arnom-ms commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+
+Wir glauben, dass _Trusted Committers_ das Anwerben und Betreuen während der Pull Requests über das Erreichen von Veröffentlichungsterminen stellen sollten, außer es sprechen triftige Gründe dagegen.
+Gute Betreuung während eines Pull Requests führt zu höherem Vertrauen und Einbindung von https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_], was im Gegenzug zu mehr Mitarbeit führt.
+Wir werden das in https://innersourcecommons.org/resources/learningpath/trusted-committer/04/["Upleveling the Community"] näher betrachten.
```
replace "das" with "diesen Aspekt"

RJPlog replied:
done

arnom-ms commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Gute Betreuung während eines Pull Requests führt zu höherem Vertrauen und Einbindung von https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_], was im Gegenzug zu mehr Mitarbeit führt.
+Wir werden das in https://innersourcecommons.org/resources/learningpath/trusted-committer/04/["Upleveling the Community"] näher betrachten.
+
+Schlussendlich gibt es Menschen, die sich in _InnerSource Communities_ engagieren, weil sie sich dort auf das Entwickeln von Software konzentrieren können, anstatt Aufgaben nachzugehen, die häufig als überflüssig oder unnötig angesehen werden.
```
Replace "Schlussendlich" with "Schliesslich"

RJPlog replied:
done

arnom-ms commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Gute Betreuung während eines Pull Requests führt zu höherem Vertrauen und Einbindung von https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_], was im Gegenzug zu mehr Mitarbeit führt.
+Wir werden das in https://innersourcecommons.org/resources/learningpath/trusted-committer/04/["Upleveling the Community"] näher betrachten.
+
+Schlussendlich gibt es Menschen, die sich in _InnerSource Communities_ engagieren, weil sie sich dort auf das Entwickeln von Software konzentrieren können, anstatt Aufgaben nachzugehen, die häufig als überflüssig oder unnötig angesehen werden.
```
add "anderen" to change the sentence to  .... "anstatt anderen Aufgaben nachzugehen..."

Remove the "," before "anstatt"

RJPlog replied:
done

RJPlog replied:
done

arnom-ms commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+
+Ein wichtiger Aspekt dieser Richtlinien ist, was wir in Pull Requests  _signaling_ nennen: Wie soll ein Kommentar aussehen? Was bedeutet es, wenn ich einen Kommentar "_like_" oder "_+1_" vergebe?
+Wie unterscheidet sich beim _@mentioning_ die Verwendung eines _/CC_-Vorzeichens von einem _/FYI_-Vorzeichen? Allgemein gesprochen müssen _Trusted Committers_ sicherstellen, dass der Prozess, im Projekt mitzuarbeiten, nicht mehr
+Probleme verursacht, sondern die Organisation unterstützt, Probleme im Projekt zu identifizeren und zu lösen. Schließlich sollten _Trusted Committers_ ihre Organisation dazu befähigen, prozessbezogene Probleme zu finden 
```
remove "," between "unterstützt" und "Probleme"

RJPlog replied:
done

arnom-ms commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+
+Ein wichtiger Aspekt dieser Richtlinien ist, was wir in Pull Requests  _signaling_ nennen: Wie soll ein Kommentar aussehen? Was bedeutet es, wenn ich einen Kommentar "_like_" oder "_+1_" vergebe?
+Wie unterscheidet sich beim _@mentioning_ die Verwendung eines _/CC_-Vorzeichens von einem _/FYI_-Vorzeichen? Allgemein gesprochen müssen _Trusted Committers_ sicherstellen, dass der Prozess, im Projekt mitzuarbeiten, nicht mehr
+Probleme verursacht, sondern die Organisation unterstützt, Probleme im Projekt zu identifizeren und zu lösen. Schließlich sollten _Trusted Committers_ ihre Organisation dazu befähigen, prozessbezogene Probleme zu finden 
```
remove trailing s in  _Trusted_Committers_

RJPlog replied:
done

arnom-ms commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Probleme verursacht, sondern die Organisation unterstützt, Probleme im Projekt zu identifizeren und zu lösen. Schließlich sollten _Trusted Committers_ ihre Organisation dazu befähigen, prozessbezogene Probleme zu finden 
+und diese so gut wie möglich selbst anzupassen und zu verbessern.
+
+Damit _Trusted Committers_ alle diese Verantwortlichkeiten erfüllen können, ist es wichtig, dass sie regelmäßig mit den Mitgliedern der Organisation kommunizieren und ein Gespür für die allgemeine Stimmung entwickeln. 
```
remove trailing s in _Trusted_Committers_

RJPlog replied:
done

arnom-ms commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Wir werden dies näher im Kapitel https://innersourcecommons.org/resources/learningpath/trusted-committer/06/["Advocating the Community's
+Needs"] betrachten.
+
+Zusammengefasst sollten _Trusted Committers_ sich bemühen, ein einladendes und annerkennendes Umfeld für ihre https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] zu erzeugen, welches 
```
remove trailing s in _Trusted_Committers_

RJPlog replied:
done

arnom-ms commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Needs"] betrachten.
+
+Zusammengefasst sollten _Trusted Committers_ sich bemühen, ein einladendes und annerkennendes Umfeld für ihre https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] zu erzeugen, welches 
+diesen erlaubt, sich auf das Schreiben von Software zu konzentrieren und ihre Persönlichkeit weiterzuentwickeln, indem sie von anderen Mitgliedern der Organisation lernen.
```
remove both ","

RJPlog replied:
done

lenucksi commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Zur Sicherstellung der Qualit�t wird am h�ufigsten das _Peer Review_ genutzt, welches normalerweise Bestandteil der Pull Requests (PR's) ist.
+ W�hrend f�r gew�hnlich jedermann Pull Requests starten und daran teilnehmen kann, indem er auf notwendige 
+Verbesserungen hinweist, ist es normalerweise nur dem _Trusted Committer_ m�glich, Beitr�ge zu akzeptieren oder abzulehnen.
+Das ist es, was wir an anderer Stelle damit gemeint haben, wenn wir davon sprachen, dass "_Trusted Committers_ n�her an der Freigabe des Codes sind". 
```
```suggestion
Das ist es, was wir an anderer Stelle damit gemeint haben, wenn wir davon sprachen, dass "_Trusted Committer_ näher an der Freigabe des Codes sind". 
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Zur Sicherstellung der Qualit�t wird am h�ufigsten das _Peer Review_ genutzt, welches normalerweise Bestandteil der Pull Requests (PR's) ist.
+ W�hrend f�r gew�hnlich jedermann Pull Requests starten und daran teilnehmen kann, indem er auf notwendige 
+Verbesserungen hinweist, ist es normalerweise nur dem _Trusted Committer_ m�glich, Beitr�ge zu akzeptieren oder abzulehnen.
+Das ist es, was wir an anderer Stelle damit gemeint haben, wenn wir davon sprachen, dass "_Trusted Committers_ n�her an der Freigabe des Codes sind". 
+ _Trusted Committers_ sollten den https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] auch w�hrend eines PR's helfen,
```
```suggestion
Das ist es, was wir an anderer Stelle damit gemeint haben, wenn wir davon sprachen, dass "_Trusted Committers_ n�her an der Freigabe des Codes sind". 
 _Trusted Committer_ sollten den https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] auch während eines PR's helfen,
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+
+Dennoch ist es letztendlich die Aufgabe des _Contributors_, dies zu erreichen.
+Es ist nicht Aufgabe eines _Trusted Committers_, alle Beitr�ge grunds�tzlich zu akzeptieren, sondern nur diejenigen, die die definierten Kriterien bez�glich Qualit�t und Umfang erf�llen. 
+Und _Trusted Committers_ sollten vermeiden, den Code eines _Contributors_ umzuschreiben, um ihn passend zu machen, selbst wenn dadurch ein h�herer Aufwand bei der Betreuung des 
```
```suggestion
_Trusted Committer_ sollten vermeiden, den Code eines _Contributors_ umzuschreiben, um ihn passend zu machen, selbst wenn dadurch ein höherer Aufwand bei der Betreuung des 
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Dennoch ist es letztendlich die Aufgabe des _Contributors_, dies zu erreichen.
+Es ist nicht Aufgabe eines _Trusted Committers_, alle Beitr�ge grunds�tzlich zu akzeptieren, sondern nur diejenigen, die die definierten Kriterien bez�glich Qualit�t und Umfang erf�llen. 
+Und _Trusted Committers_ sollten vermeiden, den Code eines _Contributors_ umzuschreiben, um ihn passend zu machen, selbst wenn dadurch ein h�herer Aufwand bei der Betreuung des 
+https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] w�hrend eines PR's entsteht. _Trusted Committers_
```
```suggestion
https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] w�hrend eines PR's entsteht. _Trusted Committer_
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+und dies langfristig die Entwicklungsgeschwindigkeit der Organisation erh�ht.
+
+Manchmal sind Anforderungen und Einschr�nkungen des Projekts nicht von vorne herein bekannt, sondern werden erst w�hrend der Entwicklung erkannt.
+_Trusted Committers_ sind auch daf�r verantwortlich, dass diese Ergebnisse erfasst und dokumentiert werden, so dass diese sowohl 
```
```suggestion
_Trusted Committer_ sind auch dafür verantwortlich, dass diese Ergebnisse erfasst und dokumentiert werden, so dass diese sowohl 
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+den https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/product-owner/01-opening-article.asciidoc[_Product Owners_] als auch den 
+https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] zug�nglich sind.
+
+Aber der Aufgabenbereich des _Trusted Committers_ in Bezug auf Qualit�t geht �ber Pull Requests hinaus. 
```
```suggestion
Der Aufgabenbereich des _Trusted Committers_ in Bezug auf Qualität geht über Pull Requests hinaus. 
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] zug�nglich sind.
+
+Aber der Aufgabenbereich des _Trusted Committers_ in Bezug auf Qualit�t geht �ber Pull Requests hinaus. 
+_Trusted Committers_ betrachten Qualit�t als strategische Komponente und stellen die Langlebigkeit der zu erstellenden Software sicher. Dies beinhaltet codeorientierte Verantwortlichkeiten im Sinne von _clean code_ bis hin zur
```
```suggestion
_Trusted Committer_ betrachten Qualität als strategische Komponente und stellen die Langlebigkeit der zu erstellenden Software sicher. Dies beinhaltet codeorientierte Verantwortlichkeiten im Sinne von _clean code_ bis hin zur
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Wahrung der konzeptionellen Integrit�t der gesamten Software.
+Dazu geh�ren auch managementorientierte Aufgaben, wie zum Beispiel sicherzustellen, dass die Organisation gen�gend Zeit f�r Pflege der Software hat, oder wenn n�tig, die Verschiebung von Bereitstellungsterminen 
+zugunsten der Verbesserung der Softwarequalit�t zu veranlassen.
+Der Effektivit�t des _Trusted Committer_ basiert stark auf der Wartbarkeit des Code's ab.
```
```suggestion
Die Effektivität des _Trusted Committers_ hängt stark auf der Wartbarkeit des Codes ab.
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+zugunsten der Verbesserung der Softwarequalit�t zu veranlassen.
+Der Effektivit�t des _Trusted Committer_ basiert stark auf der Wartbarkeit des Code's ab.
+
+Fehlt letzteres, m�ssen die _Trusted Committers_ viel ihrer wertvollen Zeit investieren, um Workarounds f�r Fehler oder anf�llige Architektur zu validieren und zu dokumentieren. Dadurch haben sich nicht 
```
```suggestion
Fehlt letzteres, müssen die _Trusted Committer_ viel ihrer wertvollen Zeit investieren, um Workarounds für Fehler oder anfällige Architektur zu validieren und zu dokumentieren. Dadurch haben sie nicht 
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Fehlt letzteres, m�ssen die _Trusted Committers_ viel ihrer wertvollen Zeit investieren, um Workarounds f�r Fehler oder anf�llige Architektur zu validieren und zu dokumentieren. Dadurch haben sich nicht 
+ausreichend Zeit, um https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] einzubeziehen und zu betreuen.
+
+Zusammengefasst ist das Sicherstellen der Produktqualit�t eine Schl�sselverantwortung eines _Trusted Committers_.
```
```suggestion
Zusammengefasst ist das Sicherstellen der Produktqualität eine Schlüsselverantwortung der _Trusted Committer_.
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+== Die Leistungsfähigkeit der Organisation erhalten
+
+In der Einführung wurde darauf hingewiesen, dass _Trusted Committers_ sowohl technikorientierte als auch organisationsorientierte Verantwortlichkeiten haben.
```
```suggestion
In der Einführung wurde darauf hingewiesen, dass _Trusted Committer_ sowohl technikorientierte als auch organisationsorientierte Verantwortlichkeiten haben.
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+
+In der Einführung wurde darauf hingewiesen, dass _Trusted Committers_ sowohl technikorientierte als auch organisationsorientierte Verantwortlichkeiten haben.
+Es ist nicht ausreichend, sich nur auf den _Code_ und den Zustand des _Codes_ zu konzentrieren. Um auf lange Sicht erfolgreich zu sein, sollten _Trusted Committer_ auch danach streben, die 
+Leistungsfähigkeit und Motivation der Organisation, welche die Software erstellt, aufrechtzuerhalten. 
```
I would probably even change to "... und Motivation der Mitglieder der Organisation die die Software erstellt, ..." as otherwise the agency is attributed to the organization which might differ from the motivation of its individual members.

RJPlog replied:
done

lenucksi commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Infolgedessen wird sich eine gute Organisation kontinuierlich selbst verbessern.
+
+Warum schließen sich https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] einer Organisation an und bleiben dabei?
+Einige tun dies, weil sie sich dem Zweck oder der Mission der Organisation anschließen. Es ist die Aufgabe des _Trusted Committers_, diesen Zweck klar zu formulieren und voranzutreiben.
```
Change to plural and not singular for the TCs here too?

RJPlog replied:
done

lenucksi commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Einige tun dies, weil sie sich dem Zweck oder der Mission der Organisation anschließen. Es ist die Aufgabe des _Trusted Committers_, diesen Zweck klar zu formulieren und voranzutreiben.
+Dessen Bedeutung wird oft nicht erkannt, aber die Pflege einer Organisation und ihrer Produkte ist tatsächlich entscheidend.
+
+Ein weiterer, offensichtlicher Grund für jemanden dabeizubleiben ist, dass er die Zusammenarbeit mit anderen Mitgliedern der Organisation genießen, einschließlich der _Trusted Committers_.
```
I'll add a gender neutral phrasing here and split the sentence into two.

```suggestion
Ein weiterer, offensichtlicher Grund für eine Person dabeizubleiben ist, dass diese die Zusammenarbeit mit den anderen Mitgliedern der Organisation genießt. Dies schließt die _Trusted Committer_ ein.
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Ein weiterer, offensichtlicher Grund für jemanden dabeizubleiben ist, dass er die Zusammenarbeit mit anderen Mitgliedern der Organisation genießen, einschließlich der _Trusted Committers_.
+Eine gedeihende Organisation ist eine, deren Mitglieder sich gegenseitig mit dem höchsten Respekt behandeln und miteinander kommunizieren.
+Beiträge werden mehr wie Geschenke oder Spenden denn als Ablenkungen behandelt, und sehr gute Beiträge (insbesondere erste) werden gelobt.
+Die Aufgabe des _Trusted Committers_ besteht in erster Linie darin, ein Vorbild für andere zu sein, analog zu einem Vorbild bezüglich der erwarteten Softwarequalität zu sein.
+Falls notwendig, sollten die _Trusted Committers_ einen Verhaltenskodex entwerfen und beschließen. Falls sich Mitglieder der Organisation entgegen den Regeln der Organisation oder gar schädigend verhalten,
```
```suggestion
Die Aufgabe der _Trusted Committer_ besteht in erster Linie darin, ein Vorbild für andere zu sein, analog zu einem Vorbild bezüglich der erwarteten Softwarequalität zu sein.
Falls notwendig, sollten die _Trusted Committer_ einen Verhaltenskodex entwerfen und beschließen. Falls sich Mitglieder der Organisation entgegen den Regeln der Organisation oder gar schädigend verhalten,
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Beiträge werden mehr wie Geschenke oder Spenden denn als Ablenkungen behandelt, und sehr gute Beiträge (insbesondere erste) werden gelobt.
+Die Aufgabe des _Trusted Committers_ besteht in erster Linie darin, ein Vorbild für andere zu sein, analog zu einem Vorbild bezüglich der erwarteten Softwarequalität zu sein.
+Falls notwendig, sollten die _Trusted Committers_ einen Verhaltenskodex entwerfen und beschließen. Falls sich Mitglieder der Organisation entgegen den Regeln der Organisation oder gar schädigend verhalten,
+ist es die Aufgabe des _Trusted Committers_, dies anzusprechen. _Trusted Committers_ sollten Möglichkeiten für Zusammenkünfte der Mitglieder bieten, dafür sorgen, sich gegenseitig kennenzulernen und auftretende
```
```suggestion
ist es die Aufgabe der _Trusted Committer_, dies anzusprechen. _Trusted Committer_ sollten Möglichkeiten für Zusammenkünfte der Mitglieder bieten, dafür sorgen, sich gegenseitig kennenzulernen und auftretende
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+
+Schlussendlich gibt es Menschen, die sich in _InnerSource Communities_ engagieren, weil sie sich dort auf das Entwickeln von Software konzentrieren können, anstatt Aufgaben nachzugehen, die häufig als überflüssig oder unnötig angesehen werden.
+Dies trifft häufig auf große Unternehmen mit einem starken Fokus auf Prozesse zu.
+Die Aufgabe des _Trusted Committers_ in diesem Zusammenhang ist es, sicherzustellen dass die https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] sich auf ihre Projekte konzentrieren können, indem sie
```
If we go for plural TCs everywhere, we should do it here too.
```suggestion
Die Aufgabe der _Trusted Committer_ in diesem Zusammenhang ist es, sicherzustellen dass die https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] sich auf ihre Projekte konzentrieren können, indem sie
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Probleme verursacht, sondern die Organisation unterstützt, Probleme im Projekt zu identifizeren und zu lösen. Schließlich sollten _Trusted Committers_ ihre Organisation dazu befähigen, prozessbezogene Probleme zu finden 
+und diese so gut wie möglich selbst anzupassen und zu verbessern.
+
+Damit _Trusted Committers_ alle diese Verantwortlichkeiten erfüllen können, ist es wichtig, dass sie regelmäßig mit den Mitgliedern der Organisation kommunizieren und ein Gespür für die allgemeine Stimmung entwickeln. 
```
```suggestion
Damit _Trusted Committer_ alle diese Verantwortlichkeiten erfüllen können, ist es wichtig, dass sie regelmäßig mit den Mitgliedern der Organisation kommunizieren und ein Gespür für die allgemeine Stimmung entwickeln. 
```

RJPlog replied:
done

lenucksi commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Wir werden dies näher im Kapitel https://innersourcecommons.org/resources/learningpath/trusted-committer/06/["Advocating the Community's
+Needs"] betrachten.
+
+Zusammengefasst sollten _Trusted Committers_ sich bemühen, ein einladendes und annerkennendes Umfeld für ihre https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] zu erzeugen, welches 
```
```suggestion
Zusammengefasst sollten _Trusted Committer_ sich bemühen, ein einladendes und annerkennendes Umfeld für ihre https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] zu erzeugen, welches 
```

RJPlog replied:
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+== Sicherstellung der Produktqualit�t
+
+Lasst uns mit der wohl am �ftesten mit der Rolle des _Trusted Committers_ in 
+Verbindung gebrachten Verantwortung beginnen: der Sicherstellung der Produktqualit�t.
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Lasst uns mit der wohl am �ftesten mit der Rolle des _Trusted Committers_ in 
+Verbindung gebrachten Verantwortung beginnen: der Sicherstellung der Produktqualit�t.
+
+In einer _InnerSource_-Organisation sind die _Trusted Committers_ verantwortlich f�r
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Verantwortlichkeit in diesem Fall bedeutet die Notwendigkeit, sicherzustellen, dass diese Entscheidungen
+respektiert werden. Das schlie�t die Kommunikation dieser Entscheidungen und - falls notwendig - deren Verfechtung
+ innerhalb und au�erhalb der Organisationseinheit mit ein. Das bedeutet aber nicht, dass infolge alle technologisch
+basierten Entscheidungen von den _Trusted Commiters_ getroffen werden m�ssen, noch dass diese die komplette 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+
+Die Aufgabe eines _Trusted Committers_ ist es, die Qualit�tsstandards innerhalb ihrer Organisation zu kommunizieren 
+und zu erl�utern, vor allem aber sie so zu formulieren, dass sie f�r ihre https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_]
+verst�ndlich und anwendbar sind. Dies beinhaltet nat�rlich auch die schriftliche Dokumentation, aber es ist f�r eine _InnerSource_-Organisation der effektivste Weg 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+verst�ndlich und anwendbar sind. Dies beinhaltet nat�rlich auch die schriftliche Dokumentation, aber es ist f�r eine _InnerSource_-Organisation der effektivste Weg 
+diese Standards vorzuleben und Beispiele zu geben. 
+Wir glauben, dass es ein erstrebenswertes Ziel f�r eine _InnerSource_-Organisation sein sollte, sich nicht nur in der Arbeitsmethodik
+ von einer klassischen Entwicklungsorganisation zu unterscheiden, sondern auch in der Qualit�t der Software, die sie entwickelt. Softwarequalit�t auf h�chstem 
```
I am not fit in grammar, but I had a review by a germanist, who added/changed about 100reds of colons and explained me the rules, so I will leave this one here :-)

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+== Sicherstellung der Produktqualit�t
+
+Lasst uns mit der wohl am �ftesten mit der Rolle des _Trusted Committers_ in 
+Verbindung gebrachten Verantwortung beginnen: der Sicherstellung der Produktqualit�t.
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Verbindung gebrachten Verantwortung beginnen: der Sicherstellung der Produktqualit�t.
+
+In einer _InnerSource_-Organisation sind die _Trusted Committers_ verantwortlich f�r
+alle technologisch basierten Entscheidungen, speziell derjenigen, die sich auf die Produktqualit�t beziehen.
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+In einer _InnerSource_-Organisation sind die _Trusted Committers_ verantwortlich f�r
+alle technologisch basierten Entscheidungen, speziell derjenigen, die sich auf die Produktqualit�t beziehen.
+Verantwortlichkeit in diesem Fall bedeutet die Notwendigkeit, sicherzustellen, dass diese Entscheidungen
+respektiert werden. Das schlie�t die Kommunikation dieser Entscheidungen und - falls notwendig - deren Verfechtung
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Niveau ist eine Grundvoraussetzung, um Vertrauen seitens des Managements und der Benutzer in eine  _InnerSource_-Organisationeinheit aufzubauen 
+und aufrecht zu erhalten. Wir alle wissen, wie schnell dieses Vertrauen durch eine einzige fehlerhafte Auslieferung ersch�ttert werden kann.
+
+_Trusted Committers_ stellen au�erdem sicher, das in der Organisation die notwendige Infrastruktur und die notwendigen Tools vorhanden sind, welche zur Entwicklung qualitativ 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+_Trusted Committers_ stellen au�erdem sicher, das in der Organisation die notwendige Infrastruktur und die notwendigen Tools vorhanden sind, welche zur Entwicklung qualitativ 
+hochwertiger Software ben�tigt werden.
+Zur Sicherstellung der Qualit�t wird am h�ufigsten das _Peer Review_ genutzt, welches normalerweise Bestandteil der Pull Requests (PR's) ist.
+ W�hrend f�r gew�hnlich jedermann Pull Requests starten und daran teilnehmen kann, indem er auf notwendige 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+hochwertiger Software ben�tigt werden.
+Zur Sicherstellung der Qualit�t wird am h�ufigsten das _Peer Review_ genutzt, welches normalerweise Bestandteil der Pull Requests (PR's) ist.
+ W�hrend f�r gew�hnlich jedermann Pull Requests starten und daran teilnehmen kann, indem er auf notwendige 
+Verbesserungen hinweist, ist es normalerweise nur dem _Trusted Committer_ m�glich, Beitr�ge zu akzeptieren oder abzulehnen.
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+ W�hrend f�r gew�hnlich jedermann Pull Requests starten und daran teilnehmen kann, indem er auf notwendige 
+Verbesserungen hinweist, ist es normalerweise nur dem _Trusted Committer_ m�glich, Beitr�ge zu akzeptieren oder abzulehnen.
+Das ist es, was wir an anderer Stelle damit gemeint haben, wenn wir davon sprachen, dass "_Trusted Committers_ n�her an der Freigabe des Codes sind". 
+ _Trusted Committers_ sollten den https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] auch w�hrend eines PR's helfen,
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+ihre Beitr�ge fertig zu stellen.
+
+Dennoch ist es letztendlich die Aufgabe des _Contributors_, dies zu erreichen.
+Es ist nicht Aufgabe eines _Trusted Committers_, alle Beitr�ge grunds�tzlich zu akzeptieren, sondern nur diejenigen, die die definierten Kriterien bez�glich Qualit�t und Umfang erf�llen. 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+
+Dennoch ist es letztendlich die Aufgabe des _Contributors_, dies zu erreichen.
+Es ist nicht Aufgabe eines _Trusted Committers_, alle Beitr�ge grunds�tzlich zu akzeptieren, sondern nur diejenigen, die die definierten Kriterien bez�glich Qualit�t und Umfang erf�llen. 
+Und _Trusted Committers_ sollten vermeiden, den Code eines _Contributors_ umzuschreiben, um ihn passend zu machen, selbst wenn dadurch ein h�herer Aufwand bei der Betreuung des 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Es ist nicht Aufgabe eines _Trusted Committers_, alle Beitr�ge grunds�tzlich zu akzeptieren, sondern nur diejenigen, die die definierten Kriterien bez�glich Qualit�t und Umfang erf�llen. 
+Und _Trusted Committers_ sollten vermeiden, den Code eines _Contributors_ umzuschreiben, um ihn passend zu machen, selbst wenn dadurch ein h�herer Aufwand bei der Betreuung des 
+https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] w�hrend eines PR's entsteht. _Trusted Committers_
+nehmen eine l�ngerfristige Perspektive ein und verstehen, dass diese Art der Unterst�tzung eine Investition in die Langlebigkeit der Organisation darstellt, 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+den https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/product-owner/01-opening-article.asciidoc[_Product Owners_] als auch den 
+https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] zug�nglich sind.
+
+Aber der Aufgabenbereich des _Trusted Committers_ in Bezug auf Qualit�t geht �ber Pull Requests hinaus. 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Wahrung der konzeptionellen Integrit�t der gesamten Software.
+Dazu geh�ren auch managementorientierte Aufgaben, wie zum Beispiel sicherzustellen, dass die Organisation gen�gend Zeit f�r Pflege der Software hat, oder wenn n�tig, die Verschiebung von Bereitstellungsterminen 
+zugunsten der Verbesserung der Softwarequalit�t zu veranlassen.
+Der Effektivit�t des _Trusted Committer_ basiert stark auf der Wartbarkeit des Code's ab.
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+zugunsten der Verbesserung der Softwarequalit�t zu veranlassen.
+Der Effektivit�t des _Trusted Committer_ basiert stark auf der Wartbarkeit des Code's ab.
+
+Fehlt letzteres, m�ssen die _Trusted Committers_ viel ihrer wertvollen Zeit investieren, um Workarounds f�r Fehler oder anf�llige Architektur zu validieren und zu dokumentieren. Dadurch haben sich nicht 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Aber der Aufgabenbereich des _Trusted Committers_ in Bezug auf Qualit�t geht �ber Pull Requests hinaus. 
+_Trusted Committers_ betrachten Qualit�t als strategische Komponente und stellen die Langlebigkeit der zu erstellenden Software sicher. Dies beinhaltet codeorientierte Verantwortlichkeiten im Sinne von _clean code_ bis hin zur
+Wahrung der konzeptionellen Integrit�t der gesamten Software.
+Dazu geh�ren auch managementorientierte Aufgaben, wie zum Beispiel sicherzustellen, dass die Organisation gen�gend Zeit f�r Pflege der Software hat, oder wenn n�tig, die Verschiebung von Bereitstellungsterminen 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+zugunsten der Verbesserung der Softwarequalit�t zu veranlassen.
+Der Effektivit�t des _Trusted Committer_ basiert stark auf der Wartbarkeit des Code's ab.
+
+Fehlt letzteres, m�ssen die _Trusted Committers_ viel ihrer wertvollen Zeit investieren, um Workarounds f�r Fehler oder anf�llige Architektur zu validieren und zu dokumentieren. Dadurch haben sich nicht 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Fehlt letzteres, m�ssen die _Trusted Committers_ viel ihrer wertvollen Zeit investieren, um Workarounds f�r Fehler oder anf�llige Architektur zu validieren und zu dokumentieren. Dadurch haben sich nicht 
+ausreichend Zeit, um https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] einzubeziehen und zu betreuen.
+
+Zusammengefasst ist das Sicherstellen der Produktqualit�t eine Schl�sselverantwortung eines _Trusted Committers_.
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Zusammengefasst ist das Sicherstellen der Produktqualit�t eine Schl�sselverantwortung eines _Trusted Committers_.
+Sie bestimmen die Qualit�tsstandards und gehen mit gutem Vorbild voran. Sie unterst�tzen bei Pull Requests und helfen den 
+https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] die Qualit�tsstandards zu erf�llen. 
+Sie �bernehmen auch die Verantwortung f�r die langfristige Wartbarkeit und Qualit�t der Software. 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Lasst uns mit der wohl am �ftesten mit der Rolle des _Trusted Committers_ in 
+Verbindung gebrachten Verantwortung beginnen: der Sicherstellung der Produktqualit�t.
+
+In einer _InnerSource_-Organisation sind die _Trusted Committers_ verantwortlich f�r
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Verantwortlichkeit in diesem Fall bedeutet die Notwendigkeit, sicherzustellen, dass diese Entscheidungen
+respektiert werden. Das schlie�t die Kommunikation dieser Entscheidungen und - falls notwendig - deren Verfechtung
+ innerhalb und au�erhalb der Organisationseinheit mit ein. Das bedeutet aber nicht, dass infolge alle technologisch
+basierten Entscheidungen von den _Trusted Commiters_ getroffen werden m�ssen, noch dass diese die komplette 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+
+Die Aufgabe eines _Trusted Committers_ ist es, die Qualit�tsstandards innerhalb ihrer Organisation zu kommunizieren 
+und zu erl�utern, vor allem aber sie so zu formulieren, dass sie f�r ihre https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_]
+verst�ndlich und anwendbar sind. Dies beinhaltet nat�rlich auch die schriftliche Dokumentation, aber es ist f�r eine _InnerSource_-Organisation der effektivste Weg 
```
done

RJPlog commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,58 @@
+Niveau ist eine Grundvoraussetzung, um Vertrauen seitens des Managements und der Benutzer in eine  _InnerSource_-Organisationeinheit aufzubauen 
+und aufrecht zu erhalten. Wir alle wissen, wie schnell dieses Vertrauen durch eine einzige fehlerhafte Auslieferung ersch�ttert werden kann.
+
+_Trusted Committers_ stellen au�erdem sicher, das in der Organisation die notwendige Infrastruktur und die notwendigen Tools vorhanden sind, welche zur Entwicklung qualitativ 
```
done

RJPlog commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+== Die Leistungsfähigkeit der Organisation erhalten
+
+In der Einführung wurde darauf hingewiesen, dass _Trusted Committers_ sowohl technikorientierte als auch organisationsorientierte Verantwortlichkeiten haben.
```
done

RJPlog commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Einige tun dies, weil sie sich dem Zweck oder der Mission der Organisation anschließen. Es ist die Aufgabe des _Trusted Committers_, diesen Zweck klar zu formulieren und voranzutreiben.
+Dessen Bedeutung wird oft nicht erkannt, aber die Pflege einer Organisation und ihrer Produkte ist tatsächlich entscheidend.
+
+Ein weiterer, offensichtlicher Grund für jemanden dabeizubleiben ist, dass er die Zusammenarbeit mit anderen Mitgliedern der Organisation genießen, einschließlich der _Trusted Committers_.
```
done

RJPlog commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Dessen Bedeutung wird oft nicht erkannt, aber die Pflege einer Organisation und ihrer Produkte ist tatsächlich entscheidend.
+
+Ein weiterer, offensichtlicher Grund für jemanden dabeizubleiben ist, dass er die Zusammenarbeit mit anderen Mitgliedern der Organisation genießen, einschließlich der _Trusted Committers_.
+Eine gedeihende Organisation ist eine, deren Mitglieder sich gegenseitig mit dem höchsten Respekt behandeln und miteinander kommunizieren.
```
done

RJPlog commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+
+Ein weiterer, offensichtlicher Grund für jemanden dabeizubleiben ist, dass er die Zusammenarbeit mit anderen Mitgliedern der Organisation genießen, einschließlich der _Trusted Committers_.
+Eine gedeihende Organisation ist eine, deren Mitglieder sich gegenseitig mit dem höchsten Respekt behandeln und miteinander kommunizieren.
+Beiträge werden mehr wie Geschenke oder Spenden denn als Ablenkungen behandelt, und sehr gute Beiträge (insbesondere erste) werden gelobt.
```
done

RJPlog commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Eine gedeihende Organisation ist eine, deren Mitglieder sich gegenseitig mit dem höchsten Respekt behandeln und miteinander kommunizieren.
+Beiträge werden mehr wie Geschenke oder Spenden denn als Ablenkungen behandelt, und sehr gute Beiträge (insbesondere erste) werden gelobt.
+Die Aufgabe des _Trusted Committers_ besteht in erster Linie darin, ein Vorbild für andere zu sein, analog zu einem Vorbild bezüglich der erwarteten Softwarequalität zu sein.
+Falls notwendig, sollten die _Trusted Committers_ einen Verhaltenskodex entwerfen und beschließen. Falls sich Mitglieder der Organisation entgegen den Regeln der Organisation oder gar schädigend verhalten,
```
done

RJPlog commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Beiträge werden mehr wie Geschenke oder Spenden denn als Ablenkungen behandelt, und sehr gute Beiträge (insbesondere erste) werden gelobt.
+Die Aufgabe des _Trusted Committers_ besteht in erster Linie darin, ein Vorbild für andere zu sein, analog zu einem Vorbild bezüglich der erwarteten Softwarequalität zu sein.
+Falls notwendig, sollten die _Trusted Committers_ einen Verhaltenskodex entwerfen und beschließen. Falls sich Mitglieder der Organisation entgegen den Regeln der Organisation oder gar schädigend verhalten,
+ist es die Aufgabe des _Trusted Committers_, dies anzusprechen. _Trusted Committers_ sollten Möglichkeiten für Zusammenkünfte der Mitglieder bieten, dafür sorgen, sich gegenseitig kennenzulernen und auftretende
```
done

RJPlog commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Konflikte einvernehmlich zu lösen.
+
+Menschen neigen auch dazu sich zu beteiligen, weil die Mitarbeit in einer _InnerSource Community_ eine hervorragende Gelegenheit bietet, neue Fähigkeiten zu erlernen und persönlich zu wachsen.
+Auch dies ist ein Aspekt, in dem die Rolle des _Trusted Committers_ wirklich wichtig ist.
```
done

RJPlog commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+ist es die Aufgabe des _Trusted Committers_, dies anzusprechen. _Trusted Committers_ sollten Möglichkeiten für Zusammenkünfte der Mitglieder bieten, dafür sorgen, sich gegenseitig kennenzulernen und auftretende
+Konflikte einvernehmlich zu lösen.
+
+Menschen neigen auch dazu sich zu beteiligen, weil die Mitarbeit in einer _InnerSource Community_ eine hervorragende Gelegenheit bietet, neue Fähigkeiten zu erlernen und persönlich zu wachsen.
```
done

RJPlog commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+
+Menschen neigen auch dazu sich zu beteiligen, weil die Mitarbeit in einer _InnerSource Community_ eine hervorragende Gelegenheit bietet, neue Fähigkeiten zu erlernen und persönlich zu wachsen.
+Auch dies ist ein Aspekt, in dem die Rolle des _Trusted Committers_ wirklich wichtig ist.
+_Trusted Committers_ sind oft Mentoren für jüngere Entwickler, und verwenden ausdrücklich Zeit während der Pull Requests, um nicht nur auf Verbesserungspotentiale hinzuweisen, sondern auch im Detail zu erklären, warum und wie etwas
```
done

RJPlog commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,50 @@
+Menschen neigen auch dazu sich zu beteiligen, weil die Mitarbeit in einer _InnerSource Community_ eine hervorragende Gelegenheit bietet, neue Fähigkeiten zu erlernen und persönlich zu wachsen.
+Auch dies ist ein Aspekt, in dem die Rolle des _Trusted Committers_ wirklich wichtig ist.
+_Trusted Committers_ sind oft Mentoren für jüngere Entwickler, und verwenden ausdrücklich Zeit während der Pull Requests, um nicht nur auf Verbesserungspotentiale hinzuweisen, sondern auch im Detail zu erklären, warum und wie etwas
+verbessert werden muss. Sie liefern die Theorie oder das Hintergrundwissen zu Verbesserungen und schlagen die besten Möglichkeiten vor, Änderungen oder Verbesserungen zu implementieren.
```
done

RJPlog commented:
thx for the reviews! I changed both files accoring to the recommendations. There are some comments about commas, I removed them even if they seem to be correct, at least in the view of somebody who know's better than me, but also for my understandig they are to much. The files schould not be read by germanists but by SW-developers, so I guess it is better to have a good readability instead of grammatical correctness :-)

MaineC commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,39 @@
+
+Die Aufgabe eines _Trusted Committer_ ist es, die Qualit�tsstandards innerhalb ihrer Organisation zu kommunizieren und zu erl�utern, vor allem und sie so zu formulieren, dass sie f�r ihre https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] verst�ndlich und anwendbar sind. 
+Dies beinhaltet nat�rlich auch die schriftliche Dokumentation, und dies ist f�r eine _InnerSource_-Organisation der effektivste Weg diese Standards vorzuleben und Beispiele zu geben. 
+Wir glauben, dass es ein erstrebenswertes Ziel f�r eine _InnerSource_-Organisation sein sollte, sich nicht nur in der Arbeitsmethodik von einer klassischen Entwicklungsorganisation zu unterscheiden, sondern auch in der Qualit�t der Software, die sie entwickelt. 
```
```suggestion
Wir glauben, dass es ein erstrebenswertes Ziel für eine _InnerSource_-Community sein sollte, sich nicht nur in der Arbeitsorganisation von klassischen Entwicklungsprojekten zu unterscheiden, sondern auch in der Qualität der Software, die sie entwickelt. 
```

MaineC commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,39 @@
+Die Aufgabe eines _Trusted Committer_ ist es, die Qualit�tsstandards innerhalb ihrer Organisation zu kommunizieren und zu erl�utern, vor allem und sie so zu formulieren, dass sie f�r ihre https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] verst�ndlich und anwendbar sind. 
+Dies beinhaltet nat�rlich auch die schriftliche Dokumentation, und dies ist f�r eine _InnerSource_-Organisation der effektivste Weg diese Standards vorzuleben und Beispiele zu geben. 
+Wir glauben, dass es ein erstrebenswertes Ziel f�r eine _InnerSource_-Organisation sein sollte, sich nicht nur in der Arbeitsmethodik von einer klassischen Entwicklungsorganisation zu unterscheiden, sondern auch in der Qualit�t der Software, die sie entwickelt. 
+Softwarequalit�t auf h�chstem Niveau ist eine Grundvoraussetzung, um Vertrauen seitens des Managements und der Benutzer in eine  _InnerSource_-Organisationeinheit aufzubauen und aufrecht zu erhalten. 
```
```suggestion
Softwarequalität auf höchstem Niveau ist eine Grundvoraussetzung, um Vertrauen seitens des Managements und der Benutzer in eine  _InnerSource_-Community aufzubauen und aufrecht zu erhalten. 
```

MaineC commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,39 @@
+Dies beinhaltet nat�rlich auch die schriftliche Dokumentation, und dies ist f�r eine _InnerSource_-Organisation der effektivste Weg diese Standards vorzuleben und Beispiele zu geben. 
+Wir glauben, dass es ein erstrebenswertes Ziel f�r eine _InnerSource_-Organisation sein sollte, sich nicht nur in der Arbeitsmethodik von einer klassischen Entwicklungsorganisation zu unterscheiden, sondern auch in der Qualit�t der Software, die sie entwickelt. 
+Softwarequalit�t auf h�chstem Niveau ist eine Grundvoraussetzung, um Vertrauen seitens des Managements und der Benutzer in eine  _InnerSource_-Organisationeinheit aufzubauen und aufrecht zu erhalten. 
+Wir alle wissen, wie schnell dieses Vertrauen durch eine einzige fehlerhafte Auslieferung ersch�ttert werden kann.
```
```suggestion
Wir alle wissen, wie schnell dieses Vertrauen durch eine einziges fehlerhaftes Release erschüttert werden kann.
```

MaineC commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,39 @@
+Softwarequalit�t auf h�chstem Niveau ist eine Grundvoraussetzung, um Vertrauen seitens des Managements und der Benutzer in eine  _InnerSource_-Organisationeinheit aufzubauen und aufrecht zu erhalten. 
+Wir alle wissen, wie schnell dieses Vertrauen durch eine einzige fehlerhafte Auslieferung ersch�ttert werden kann.
+
+_Trusted Committer_ stellen au�erdem sicher, das in der Organisation die notwendige Infrastruktur und die notwendigen Tools vorhanden sind, welche zur Entwicklung qualitativ hochwertiger Software ben�tigt werden.
```
```suggestion
_Trusted Committer_ stellen außerdem sicher, das die Community die notwendige Infrastruktur und die notwendigen Tools zur Verfügung stehen, die zur Entwicklung qualitativ hochwertiger Software benötigt werden.
```

MaineC commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,39 @@
+_Trusted Committer_ stellen au�erdem sicher, das in der Organisation die notwendige Infrastruktur und die notwendigen Tools vorhanden sind, welche zur Entwicklung qualitativ hochwertiger Software ben�tigt werden.
+Zur Sicherstellung der Qualit�t wird am h�ufigsten das _Peer Review_ genutzt, welches normalerweise Bestandteil der Pull Requests(PR's) ist.
+W�hrend f�r gew�hnlich jedermann Pull Requests starten und daran teilnehmen kann indem er auf notwendige Verbesserungen hinweist, ist es normalerweise nur dem _Trusted Committer_ m�glich Beitr�ge zu akzeptieren oder abzulehnen.
+Das ist es, was wir an anderer Stelle damit gemeint haben, wenn wir davon sprachen, dass "_Trusted Committer_ n�her an der Freigabe des Codes sind". 
```
```suggestion
Das ist es, was wir an anderer Stelle damit gemeint haben, wenn wir davon sprachen, dass "_Trusted Committer_ Code in Produktion bringen können". 
```

MaineC commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,39 @@
+
+Dennoch ist es letztendlich die Aufgabe des _Contributors_, dies zu erreichen.
+Es ist nicht Aufgabe eines _Trusted Committer_, alle Beitr�ge grunds�tzlich zu akzeptieren, sondern nur diejenigen, die die definierten Kriterien bez�glich Qualit�t, Umfang und Projektausrichtung erf�llen. 
+_Trusted Committer_ sollten vermeiden, den Code eines _Contributors_ umzuschreiben, um ihn passend zu machen, selbst wenn dadurch ein h�herer Aufwand bei der Betreuung des https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] w�hrend eines PR's entsteht. 
```
```suggestion
_Trusted Committer_ sollten vermeiden, den Code eines _Contributors_ selbst umzuschreiben, selbst wenn dadurch ein höherer Aufwand bei der Unterstützung des https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] bei einem PR anfällt. 
```

MaineC commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,39 @@
+Dennoch ist es letztendlich die Aufgabe des _Contributors_, dies zu erreichen.
+Es ist nicht Aufgabe eines _Trusted Committer_, alle Beitr�ge grunds�tzlich zu akzeptieren, sondern nur diejenigen, die die definierten Kriterien bez�glich Qualit�t, Umfang und Projektausrichtung erf�llen. 
+_Trusted Committer_ sollten vermeiden, den Code eines _Contributors_ umzuschreiben, um ihn passend zu machen, selbst wenn dadurch ein h�herer Aufwand bei der Betreuung des https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] w�hrend eines PR's entsteht. 
+_Trusted Committer_ nehmen eine l�ngerfristige Perspektive ein und verstehen, dass diese Art der Unterst�tzung eine Investition in die Langlebigkeit der Organisation darstellt und dies langfristig die Entwicklungsgeschwindigkeit der Organisation erh�ht.
```
```suggestion
_Trusted Committer_ nehmen eine längerfristige Perspektive ein und verstehen, dass diese Art der Unterstützung eine Investition in die Langlebigkeit der Community darstellt und dies langfristig die Entwicklungsgeschwindigkeit der Community erhöht.
```

MaineC commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,39 @@
+_Trusted Committer_ sind auch daf�r verantwortlich, dass diese Ergebnisse erfasst und dokumentiert werden, so dass diese sowohl den https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/product-owner/01-opening-article.asciidoc[_Product Owners_] als auch den https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_ ...
+
+Der Aufgabenbereich des _Trusted Committer_ in Bezug auf Qualit�t geht �ber Pull Requests hinaus. 
+_Trusted Committer_ betrachten Qualit�t als strategische Komponente und stellen die Langlebigkeit der zu erstellenden Software sicher. Dies beinhaltet codeorientierte Verantwortlichkeiten im Sinne von _clean code_ bis hin zur Wahrung der konzeptionellen Integrit�t der gesamten Software.
```
I'm not sure, the original text of "cleanliness of the code" should be translated to "clean code" - the latter one means a very specific type of understanding of what cleanliness of the code means that I'm not sure we want to push for here. Maybe translate with something like "Sauberkeit des Quellcodes" or "Verständlichkeit des Quellcodes"

MaineC commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,39 @@
+
+Der Aufgabenbereich des _Trusted Committer_ in Bezug auf Qualit�t geht �ber Pull Requests hinaus. 
+_Trusted Committer_ betrachten Qualit�t als strategische Komponente und stellen die Langlebigkeit der zu erstellenden Software sicher. Dies beinhaltet codeorientierte Verantwortlichkeiten im Sinne von _clean code_ bis hin zur Wahrung der konzeptionellen Integrit�t der gesamten Software.
+Dazu geh�ren auch managementorientierte Aufgaben wie zum Beispiel sicherzustellen, dass die Organisation gen�gend Zeit f�r Pflege der Software hat, oder wenn n�tig, die Verschiebung von Bereitstellungsterminen zugunsten der Verbesserung der Softwarequalit�t zu veranlassen.
```
```suggestion
Dazu gehören auch managementorientierte Aufgaben wie zum Beispiel sicherzustellen, dass die Community genügend Zeit für Refaktorierung der Software hat, oder wenn nötig, die Verschiebung von Releaseterminen zugunsten der Verbesserung der Softwarequalität zu veranlassen.
```

MaineC commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,39 @@
+Der Aufgabenbereich des _Trusted Committer_ in Bezug auf Qualit�t geht �ber Pull Requests hinaus. 
+_Trusted Committer_ betrachten Qualit�t als strategische Komponente und stellen die Langlebigkeit der zu erstellenden Software sicher. Dies beinhaltet codeorientierte Verantwortlichkeiten im Sinne von _clean code_ bis hin zur Wahrung der konzeptionellen Integrit�t der gesamten Software.
+Dazu geh�ren auch managementorientierte Aufgaben wie zum Beispiel sicherzustellen, dass die Organisation gen�gend Zeit f�r Pflege der Software hat, oder wenn n�tig, die Verschiebung von Bereitstellungsterminen zugunsten der Verbesserung der Softwarequalit�t zu veranlassen.
+Die Effektivit�t des _Trusted Committers_ h�ngt stark von der Wartbarkeit des Code's ab.
```
```suggestion
Die Effektivität des _Trusted Committers_ hängt stark von der Wartbarkeit des Codes ab.
```

MaineC replied:
```suggestion
Die Effektivität des _Trusted Committers_ hängt stark von der Wartbarkeit des Codes ab.
```

MaineC commented on trusted-committer/de/02-ensuring-product-quality-de.asciidoc:
```diff
@@ -0,0 +1,39 @@
+Dazu geh�ren auch managementorientierte Aufgaben wie zum Beispiel sicherzustellen, dass die Organisation gen�gend Zeit f�r Pflege der Software hat, oder wenn n�tig, die Verschiebung von Bereitstellungsterminen zugunsten der Verbesserung der Softwarequalit�t zu veranlassen.
+Die Effektivit�t des _Trusted Committers_ h�ngt stark von der Wartbarkeit des Code's ab.
+
+Fehlt letzteres, m�ssen die _Trusted Committer_ viel ihrer wertvollen Zeit investieren um Workarounds f�r Fehler oder anf�llige Architektur zu validieren und zu dokumentieren. Dadurch haben sie nicht ausreichend Zeit, um https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] einzubeziehen und zu betreuen.
```
```suggestion
Fehlt letzteres, müssen die _Trusted Committer_ viel ihrer wertvollen Zeit investieren um Workarounds für Fehler oder anfällige Architektur zu validieren und zu dokumentieren. Dadurch haben sie nicht ausreichend Zeit, um https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_]  mit Onboarding und Mentoring zu unterstützen.
```

MaineC commented:
I went throught the first document. Some general comments:

First and most important of all: Thanks for taking the effort of translating this part of the learning path. It's amazing to see the content getting localized!

Some things that were weird to read for me: I would leave community as community instead of translating it to Organisation - the latter to me sounds more like the entire company or a rigid department in a company while InnerSource communities to me feel much more fluid and flexible.

I'd also leave things like release and ownership in their original terms, unless I'm entirely mistaken I have heard the term ownership in German agile literature as well.

One weird thing: GitHub seems to break the Umlaut letters when switching to review suggestions. Did anyone else have that issue, or is that just me on a Ubuntu machine using Firefox?

MaineC commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,42 @@
+Beiträge werden mehr als Geschenke oder Spenden denn als Ablenkungen behandelt, und sehr gute Beiträge (insbesondere erste) werden gelobt. 
+Die Aufgabe der _Trusted Committer_ besteht in erster Linie darin, ein Vorbild für andere zu sein, analog zu einem Vorbild bezüglich der erwarteten Softwarequalität zu sein. 
+Falls notwendig, sollten die _Trusted Committer_ einen Verhaltenskodex entwerfen und beschließen. 
+Falls sich Mitglieder der Organisation entgegen den Regeln der Organisation oder gar schädigend verhalten, ist es die Aufgabe der _Trusted Committer_, dies anzusprechen. _Trusted Committer_ sollten Möglichkeiten für Zusammenkünfte der Mitglieder bieten, dafür sorgen, sich gegenseitig kennenzulernen und auftretende Konflikte einvernehmlich zu lösen.
```
```suggestion
Falls sich Mitglieder der Community schädigend verhalten, ist es die Aufgabe der _Trusted Committer_, dies zu adressieren. _Trusted Committer_ sollen regelmäßige Treffen für Mitglieder anbieten (entweder persönlich oder virtuell). Dabei können sich Teilnehmergegenseitig kennenlernen und auftretende Konflikte einvernehmlich zu lösen sobald sie auftreten.
```

MaineC commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,42 @@
+Menschen neigen auch dazu sich an Projekten zu beteiligen, weil die Mitarbeit in einer _InnerSource Community_ eine hervorragende Gelegenheit bietet neue Fähigkeiten zu erlernen und als Person zu wachsen. 
+Dies ist ein weiterer Aspekt, in dem die Rolle des _Trusted Committers_ wirklich wichtig ist. 
+_Trusted Committer_ sind oft Mentoren für jüngere Entwickler, und verwenden Zeit während der Pull Requests, um nicht nur auf Verbesserungspotentiale hinzuweisen, sondern auch im Detail zu erklären, warum und wie etwas verbessert werden muss. 
+Sie liefern die Theorie oder das Hintergrundwissen zu Verbesserungen und schlagen die besten Möglichkeiten vor Änderungen oder Verbesserungen zu implementieren. Dadurch können _Trusted Committer_ die Lerngeschwindigkeit ihrer Organisation weit über die traditioneller Software-Entwicklungsorganisationen erhöhen.
```
```suggestion
Sie liefern die Theorie oder das Erfahrung bei Quellcodeänderungen und schlagen Möglichkeiten wie Änderungen am besten implementiert werden können. Dadurch können _Trusted Committer_ die Lerngeschwindigkeit ihrer Community weit über die traditioneller Software-Entwicklung erhöhen.
```

MaineC commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,42 @@
+_Trusted Committer_ sind oft Mentoren für jüngere Entwickler, und verwenden Zeit während der Pull Requests, um nicht nur auf Verbesserungspotentiale hinzuweisen, sondern auch im Detail zu erklären, warum und wie etwas verbessert werden muss. 
+Sie liefern die Theorie oder das Hintergrundwissen zu Verbesserungen und schlagen die besten Möglichkeiten vor Änderungen oder Verbesserungen zu implementieren. Dadurch können _Trusted Committer_ die Lerngeschwindigkeit ihrer Organisation weit über die traditioneller Software-Entwicklungsorganisationen erhöhen.
+
+Wir glauben, dass _Trusted Committer_ das Anwerben und Betreuen während der Pull Requests über das Erreichen von Veröffentlichungsterminen stellen sollten, außer es sprechen triftige Gründe dagegen. 
```
```suggestion
Wir glauben, dass _Trusted Committer_ das Onboarding und Mentoring während der Pull Requests über das Erreichen von Releaseterminen stellen sollten, außer es sprechen triftige Gründe dagegen. 
```

MaineC commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,42 @@
+Sie liefern die Theorie oder das Hintergrundwissen zu Verbesserungen und schlagen die besten Möglichkeiten vor Änderungen oder Verbesserungen zu implementieren. Dadurch können _Trusted Committer_ die Lerngeschwindigkeit ihrer Organisation weit über die traditioneller Software-Entwicklungsorganisationen erhöhen.
+
+Wir glauben, dass _Trusted Committer_ das Anwerben und Betreuen während der Pull Requests über das Erreichen von Veröffentlichungsterminen stellen sollten, außer es sprechen triftige Gründe dagegen. 
+Gute Betreuung während eines Pull Requests führt zu höherem Vertrauen und Einbindung von https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_], was im Gegenzug zu mehr Mitarbeit führt. 
```
```suggestion
Gute Betreuung während eines Pull Requests führt zu höherem Vertrauen und Bindung von https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_], was im Gegenzug zu mehr Mitarbeit führt. 
```

MaineC commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,42 @@
+
+Schliesslich gibt es Menschen, die sich in _InnerSource Communities_ engagieren, weil sie sich dort auf das Entwickeln von Software konzentrieren können anstatt anderen Aufgaben nachzugehen, die häufig als überflüssig oder unnötig angesehen werden. 
+Dies trifft häufig auf große Unternehmen mit einem starken Fokus auf Prozesse zu. 
+Die Aufgabe der _Trusted Committer_ in diesem Zusammenhang ist es, sicherzustellen dass die https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] sich auf ihre Projekte konzentrieren können, indem sie sinnvolle Richtlinien für die Mitarbeit kommunizieren und einsetzen.
```
```suggestion
Die Aufgabe der _Trusted Committer_ in diesem Zusammenhang ist es, sicherzustellen dass die https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] sich auf ihre Projekte konzentrieren können, indem sie sinnvolle Contribution Guidelines kommunizieren und einsetzen.
```

MaineC commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,42 @@
+Dies trifft häufig auf große Unternehmen mit einem starken Fokus auf Prozesse zu. 
+Die Aufgabe der _Trusted Committer_ in diesem Zusammenhang ist es, sicherzustellen dass die https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] sich auf ihre Projekte konzentrieren können, indem sie sinnvolle Richtlinien für die Mitarbeit kommunizieren und einsetzen.
+
+Ein wichtiger Aspekt dieser Richtlinien ist, was wir in Pull Requests  _signaling_ nennen: Wie soll ein Kommentar aussehen? 
```
```suggestion
Ein wichtiger Aspekt dieser Guidelines ist, was wir in Pull Requests  _signaling_ nennen: Wie soll ein Kommentar aussehen? 
```

MaineC commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,42 @@
+
+Ein wichtiger Aspekt dieser Richtlinien ist, was wir in Pull Requests  _signaling_ nennen: Wie soll ein Kommentar aussehen? 
+Was bedeutet es, wenn ich einen Kommentar "_like_" oder "_+1_" vergebe? Wie unterscheidet sich beim _@mentioning_ die Verwendung eines _/CC_-Vorzeichens von einem _/FYI_-Vorzeichen? 
+Allgemein gesprochen müssen _Trusted Committer_ sicherstellen, dass der Prozess, im Projekt mitzuarbeiten, nicht mehr Probleme verursacht, sondern die Organisation unterstützt Probleme im Projekt zu identifizeren und zu lösen. 
```
```suggestion
Allgemein gesprochen müssen _Trusted Committer_ sicherstellen, dass der Prozess, im Projekt mitzuarbeiten, nicht mehr Probleme verursacht, sondern die Community unterstützt Probleme im Projekt zu identifizeren und zu lösen. 
```

MaineC commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,42 @@
+Ein wichtiger Aspekt dieser Richtlinien ist, was wir in Pull Requests  _signaling_ nennen: Wie soll ein Kommentar aussehen? 
+Was bedeutet es, wenn ich einen Kommentar "_like_" oder "_+1_" vergebe? Wie unterscheidet sich beim _@mentioning_ die Verwendung eines _/CC_-Vorzeichens von einem _/FYI_-Vorzeichen? 
+Allgemein gesprochen müssen _Trusted Committer_ sicherstellen, dass der Prozess, im Projekt mitzuarbeiten, nicht mehr Probleme verursacht, sondern die Organisation unterstützt Probleme im Projekt zu identifizeren und zu lösen. 
+Schließlich sollten _Trusted Committer_ ihre Organisation dazu befähigen, prozessbezogene Probleme zu finden und diese so gut wie möglich selbst anzupassen und zu verbessern.
```
```suggestion
Schließlich sollten _Trusted Committer_ ihre Community dazu befähigen, prozessbezogene Probleme zu finden und diese so gut wie möglich selbst anzupassen und zu verbessern.
```

MaineC commented on trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc:
```diff
@@ -0,0 +1,42 @@
+Allgemein gesprochen müssen _Trusted Committer_ sicherstellen, dass der Prozess, im Projekt mitzuarbeiten, nicht mehr Probleme verursacht, sondern die Organisation unterstützt Probleme im Projekt zu identifizeren und zu lösen. 
+Schließlich sollten _Trusted Committer_ ihre Organisation dazu befähigen, prozessbezogene Probleme zu finden und diese so gut wie möglich selbst anzupassen und zu verbessern.
+
+Damit _Trusted Committer_ alle diese Verantwortlichkeiten erfüllen können, ist es wichtig, dass sie regelmäßig mit den Mitgliedern der Organisation kommunizieren und ein Gespür für die allgemeine Stimmung entwickeln. 
```
```suggestion
Damit _Trusted Committer_ alle diese Verantwortlichkeiten erfüllen können, ist es wichtig, dass sie regelmäßig mit den Mitgliedern der Community kommunizieren und ein Gespür für die allgemeine Stimmung entwickeln. 
```

RJPlog commented:
Thanks MaineC for the review, it took a while until I had time to look through due to some heavy load in the office and also the "Freibad" season :-). I will keep the spirit of your suggestions in the next rounds.

lenucksi commented:
> Thanks MaineC for the review, it took a while until I had time to look through due to some heavy load in the office and also the "Freibad" season :-). I will keep the spirit of your suggestions in the next rounds.

Looks like all review comments have been addressed  - I'm happy to merge if you don't see anything blocking this. 
If @arnom-ms wants to take a look he's also invited to do so.

arnom-ms commented:
This looks great , thank you. One small nit: The use of "organization" and "community" in the two articles is not quite consistent, and I would suggest to go through and align the use of the term.

RJPlog commented:
I just went again to all comments of chapter 02. Hopefully I have adapted now every comment. 03 is open, hope to close this soon

RJPlog commented:
Now I went also through all comments of chapter 03. Every comment should be considered now.

lenucksi reviewed (approved):
Thanks a lot for all the continuous work to factor in all the suggestions and requests, @RJPlog. This looks great right now and hence ab die Post damit!
